keyboards/ungodly/launch_pad/keymaps/default/config.h

```diff
@@ -0,0 +1,20 @@
+/* Copyright 2020 Ungodly Design <hello@ungodly.design>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#ifdef RGB_MATRIX_ENABLE
+#    define RGB_MATRIX_KEYPRESSES
```

Comment:
would prefer that this went into the keyboard's config, so that people using the configurator can use it, if it's possible.

Reply:
Dunzo. Thanks for the help.